tests/.bazelversion

```diff
@@ -1 +1 @@
-7.4.1
+8.0.0
```

Comment:
Same though there about upgrading to `8.1.0`

Reply:
Done!